components/list/list-item-nav-button-mixin.js

```diff
@@ -25,25 +25,19 @@ export const ListItemNavButtonMixin = superclass => class extends ListItemButton
 				width: 100%;
 			}
 			:host([current]) [slot="outside-control-container"] {
-				border: 3px solid var(--d2l-color-celestine);
+				background-color: var(--d2l-color-regolith);
+				border: 3px solid var(--d2l-color-celestine) !important;
 				margin-block: -1px;
 			}
-			:host([_focusing-primary-action]:not([current])) [slot="outside-control-container"] {
-				border: 2px solid var(--d2l-color-celestine);
```

Comment:
The border will be only around the list item text on focus, which is the default behaviour.